pkg/iac-providers/cft/v1/testdata/deploy.yaml

```diff
@@ -0,0 +1,30 @@
+Resources:
```

Comment:
Why the content of S3 resource is here? It should be part of KaiMonkey, isn't it? or I am missing on something here?

Reply:
Its for load file test